client/router.js

```diff
@@ -69,7 +69,7 @@ let router = new Router({
   }, {
     path: '/system-settings',
     component: SystemSettings,
-    meta: { auth: true },
+    meta: { auth: true, restricted: true },
```

Comment:
:bulb: I would either call this `require/ensureAdmin` or refactor it to accept array of allowed roles.